src/faq.md

```diff
@@ -447,6 +447,51 @@ The reference runner and several other CWL implementations support running
 those Docker format containers using the Singularity engine. Directly
 specifying a Singularity format container is not part of the CWL standards.
 
+## How does glob work when describing output values?
+The field outputBinding describes how to set the value of each output parameter. The 'glob' field specifies a pattern to find files/directories relative to the output directory. The pattern used for glob' must be either relative to the output directory, an absolute path to the output directory, or an absolute path to an input file.
+
+CWL uses the POSIX glob(3) pathname matching. Wildcards are allowed in the glob field and are useful when If you don’t know the exact name of a file or directory in advance. The wildcard characters can either be an asterisk *, a question mark matching pathnames in directories. `?` or a range `[]`.
+
+If an array used in the glob field, any files that match any pattern in the array are returned.
+
+In the example below, the glob field is used to specify the file(`hello.txt`) in which the output should be stored.
+
+```cwl
+outputs:
+  example_out:
+    type: File
+    outputBinding:
+      glob: "hello.txt"
+```
+
+The glob field in the above output section can also be written with the `*` wildcard character as
+
+```cwl
+outputs:
+  example_out:
+    type: File
+    outputBinding:
+      glob: '*.txt'
+```
+
+Glob can also be used to return all files in the output directory. For example
+
+```
+  outputs:
```

Comment:
I don't think this is proper syntax --

I don't think you can write glob in a single line with outputBinding ....

(1) you don't have a variable name in the first one and if it is going to be multiple Directories if should be a Directory array which you can either do by defining an array of type Directory or just doing Directory[]

(2)  and in the second one it seems like you also didn't define the name of the output variable again.

So below is an example that works, using that example see if  you can make one that works and then test it using the cwl validate option and then to be extra careful run the code...

cwlVersion: v1.0
class: CommandLineTool
baseCommand: [tar, --extract]
inputs:
  tarfile:
    type: File
    inputBinding:
      prefix: --file
outputs:
   untarfile:
    type: File[]
    outputBinding:
     glob: "*"

In this case we only generate one file so you don't need the outputBinding for all files but this is just an example. 




My suggestion would be to validate your examples using cwltool before putting them in for example I took the tar example and added your first code and it gave me this ...
Tool definition failed validation:
test.cwl:3:1:     Object `test.cwl` is not valid because
                  tried `CommandLineTool` but
test.cwl:11:1:       the `outputs` field is not valid because
test.cwl:13:5:         item is invalid because
test.cwl:13:22:         invalid field `glob`, expected one of: 'label', 'secondaryFiles',
                        'streamable', 'doc', 'id', 'outputBinding', 'format', 'type'